docs/extract_configmap.md

```diff
@@ -0,0 +1,47 @@
+# Creating a Configmap
+Sometimes it is advantageous to add files to a configmap to override those which are built into the image.  This is useful for specific customizations that are not provided with the product or when it is possible to resolve issues unrelated to compiled code changes.  Following is an example of how one might extract the Elasticsearch index templates to a configmap in order to provide an alias to each index.
+
+**NOTE:**  The steps which add volumes to each deploymentconfig will need to be performed after each upgrade since they are not preserved by the installation and upgrade process.
+
+1. Retrieve the source templates from https://github.com/openshift/origin-aggregated-logging/tree/release-1.5/elasticsearch/index_templates
+
+  ```
+  $ wget https://raw.githubusercontent.com/openshift/origin-aggregated-logging/release-1.5/elasticsearch/index_templates/com.redhat.viaq-openshift-project.template.json
+  $ wget https://raw.githubusercontent.com/openshift/origin-aggregated-logging/release-1.5/elasticsearch/index_templates/com.redhat.viaq-openshift-operations.template.json
+  ```
+2. Edit each file to provide the changes (e.g. alias all indicies)
+  ```
+  {
+  "aliases": {
+    ".all" : {}
+  },
+  "mappings": {
+    "_default_": {
+      "_meta": {
+        "version": "2016.10.12.0"
+      },
+      "date_detection": false,
+      "dynamic_templates": [
+        {
+    ...
+  ```
+  3. Create the configmap
+```
+$ oc create -n $LOGGING_NAMESPACE configmap index-templates --from-file=$DIR
```

Comment:
where `$LOGGING_NAMESPACE` is typically `openshift-logging`, or `logging` for older releases (use `oc get projects` to see which one you have - if you have both, use `openshift-logging`)

EXPERTS: you can also use this method to add your own Elasticsearch index templates - add your own index template .json files to `$DIR`.